js/src/forum/components/MarkdownToolbar.js

```diff
@@ -6,9 +6,12 @@ export default class MarkdownToolbar extends Component {
   oncreate(vnode) {
     super.oncreate(vnode);
 
-    const field = document.getElementById(this.attrs.for);
+    // We need to give enough time for the editor driver to instantiate.
+    setTimeout(() => {
+      const field = document.getElementById(this.attrs.for);
 
-    field.addEventListener('keydown', this.shortcut.bind(this));
+      field.addEventListener('keydown', this.shortcut.bind(this));
+    }, 25);
```

Comment:
This is all a bit arbitrary, but I'm not sure of a better place we could put this now that the actual textarea in core is added dynamically by JS

Reply:
We could also try to insert this as an input listener but that gets messy since that needs the component instance, which we won't have. So we'd need to do some tricky stuff where we pass a callback to input listeners, and another to the one here which hacks the local scope of the first one to insert this initialization.

Reply:
Actually, input listeners won't work, since they rely on events. And for that reason, I think it might be better to extend BasicEditorDriver to add the listener there.

Reply:
Fixed!

Reply:
Great! That's definitely better than the timeout.